### PR TITLE
Add GitHub support document

### PR DESCRIPTION
**Pull request by mkaz**
## Description

Adds a support page with links to relevant info.

Closers #26758

## Types of changes

New GitHub documentation page.

### CODE REVIEW

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+# Gutenberg Project Support
+
+Welcome to WordPress' Gutenberg project! We hope you join us in creating the future platform for publishing; all are welcome here.
```
Comment (annezazu) [inline .github/SUPPORT.md:4]: The first sentence reads a bit awkwardly right now so I might rephrase particularly for the sake of non English speakers.  Ideas: 

- "Welcome to the Gutenberg Project, a focus of the wider WordPress project." 
- "Welcome to Gutenberg, a WordPress project."

File: .github/SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+* Please see the [Contributing Guidelines](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md) for additional information on how to contribute.
+
+* As with all WordPress projects, we want to ensure a welcoming environment for everyone. With that in mind, all contributors are expected to follow our [Code of Conduct](https://github.com/WordPress/gutenberg/blob/master/CODE_OF_CONDUCT.md).
+
+* Join us on Slack for real-time communication, it is where maintainers coordinate around the project. To get started using Slack, see: https://make.wordpress.org/chat/
+
+* For WordPress support see the [WordPress.org support forums](https://wordpress.org/support/) — it is highly active and well maintained.
```
Comment (annezazu) [inline .github/SUPPORT.md:12]: I might say: "For general WordPress support with the core editor, see..."

**Comment by annezazu** [comment]
Left two non blocking comments. Thanks for doing this :D

File: .github/SUPPORT.md (hunk 1 above)
Comment (mkaz) [reply .github/SUPPORT.md]: I like the second one better, 👍
I grabbed it from the [CONTRIBUTING.md page](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md), so we may want to update there too.

File: .github/SUPPORT.md (hunk 2 above)
Comment (mkaz) [reply .github/SUPPORT.md]: Updated.

**Review by gziolo — APPROVED** [review]
Nice 👍

**Pull request merged**